scripts/stickerbom.py

```diff
@@ -455,9 +465,9 @@ def get_args():
                         help="Width of a label (mm).")
     parser.add_argument("--label_height", type=int, default=63.5,
                         help="Height of a label (mm).")
-    parser.add_argument("--labels-x", type=int, default=4,
+    parser.add_argument("--labels-x", type=int, default=3,
                         help="Number of columns of labels on a page.")
-    parser.add_argument("--labels-y", type=int, default=3,
+    parser.add_argument("--labels-y", type=int, default=2,
```

Comment:
You should probably avoid altering the default numbers of labels without changing the text stating what sort of label sheet the defaults are set for.